### PR TITLE
Qbe dev

### PR DESCRIPTION
**Pull request by BojanSovticEngIT**
QBE development (for Testlink)

### CODE REVIEW

File: knowage-vue/src/modules/qbe/QBE.vue [hunk 1]
```diff
@@ -0,0 +1,918 @@
...
+    height: 25%;
+}
+
+.olap-scroll-panel .p-scrollpanel-content {
+    padding: 0 !important;
+}
+.olap-scroll-panel .p-scrollpanel-bar {
+    background-color: #43749eb6;
+    width: 5px;
+}
```
Comment (Redjaw) [inline knowage-vue/src/modules/qbe/QBE.vue:900]: Can those classes be scoped? Why there are some olap scroll related classes?

**Review by Redjaw — CHANGES_REQUESTED** [review]

File: knowage-vue/src/modules/qbe/qbeTables/qbeSmartTable/QBESmartTable.vue [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+                color: #bbd6ed;
+            }
+        }
+    }
+}
+.alias-dialog .p-dialog-header,
+.alias-dialog .p-dialog-content {
+    padding: 0;
+    margin: 0;
+}
```
[the remark this answers is not in the document]
Comment (dbulatovicx32) [reply knowage-vue/src/modules/qbe/qbeTables/qbeSmartTable/QBESmartTable.vue:245]: That is a dialog class, its not part of the qbe-smart-table, i renamed it so its more clear.
I also tried creating a kn class for it, but i just cant seem to hit the styles i need with :deep, thats why its left like this

[the remark this answers is not in the document]
Comment (Redjaw) [reply knowage-vue/src/modules/qbe/qbeTables/qbeSmartTable/QBESmartTable.vue:245]: ok thx

File: knowage-vue/src/modules/qbe/QBE.vue (hunk 1 above)
Comment (dbulatovicx32) [reply knowage-vue/src/modules/qbe/QBE.vue]: Most classes cannot be scoped or targeted with :deep, but i simplified them.
What i could i moved to the descriptor and what could be connected i did so and put them in a same class.
If this is satisfying, we have finished all changes and PR can be accepted!

**Review by Redjaw — APPROVED** [review]

**Pull request merged**